### DISCUSSION TITLE
Having issue in plugins

### DISCUSSION THREAD

**Discussion by KarthikModelon** [mkdocs/mkdocs#2890] [Q&A]
Hi, I am getting an error while adding plugins to the yaml file
![image](https://user-images.githubusercontent.com/109127978/179208596-8213cce5-594b-49bd-8d62-16fc1bd8a249.png)
![image](https://user-images.githubusercontent.com/109127978/179208866-6ed7e011-6cb7-4835-9d79-3366817da43d.png)

**Comment by mur4d1n** [comment]
You should try insert a colon after `search`

**Comment by pawamoy** [comment]
You have to indent the lines below `mike`.
